vault/templates.topic.config.property.ref.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 6x0mjpmd9lollhu3umzhsy8
+title: Config Properties Reference
+desc: ''
+updated: 1646440609054
+created: 1646436664403
+---
+
+## Fields
+
+### type
+what type is this configuration
+
+### default
+the default value, if any
+
+### required
+whether this property is required
+
+### noteOverride
```

Comment:
One could argue that this note sufficiently explains what we mean by `noteOverride`, but when looking at each config notes, it's not apparent and a bit confusing what it means.

Can this be instead added in each of the overridable config's description, and also have an example usage at the bottom?